.prettierrc.json

```diff
@@ -1,10 +1,9 @@
 {
   "$schema": "https://json.schemastore.org/prettierrc",
+  "printWidth": 120,
   "semi": false,
-  "tabWidth": 2,
+  "singleAttributePerLine": true,
   "singleQuote": true,
-  "printWidth": 100,
   "trailingComma": "none",
```

Comment:
dbildungs-iam-server uses `semi: true`, `trailingComma: 'all'` and `endOfLine: 'lf'`, is there any reason for the discrepancy?

Reply:
`endOfLine` is 'lf' by default, so it's not necessary to specify.

`semi: true` and `trailingComma: 'all'` reduce readability in my opinion.

Reply:
For the `endOfline` comment: ok!

As for the second part:
I'm okay with that decision, although I would not call that "sensibly justified". Especially the missing semicolon is a big deviation that *could* lead to subtle bugs due to ASI.

Any comments @pkleybolte  ?

Reply:
done

Reply:
I definitely like to have the semicolon. I don't really care about the trailing comma.
But in the end I would leave the decision to @jaggarnaut . He ist our designated frontend developer. So he has to work with it the most by far. And it isn't a decision that is keeping us from deciding otherwise in the future.
If we decide work in a more fullstack mode and more (backend) devs are doing frontend work, I would reevaluate this decision.